alarm-datastore-cassandra/src/main/resources/schema-cassandra.cql

```diff
@@ -20,6 +20,53 @@ CREATE TABLE IF NOT EXISTS tip_wlan_keyspace.alarm (
 -- this is needed to support retrieval of all alarms for a customer
 CREATE INDEX IF NOT EXISTS idx_alarm_customerId ON tip_wlan_keyspace.alarm (customerId);
 
+CREATE TABLE IF NOT EXISTS tip_wlan_keyspace.alarm_by_acknowledged (
+    customerId int,
+    equipmentId bigint,
+    alarmCode int, 
+    createdTimestamp bigint,   
+    acknowledged boolean,
+    
+    PRIMARY KEY ((customerId, acknowledged), equipmentId, alarmCode, createdTimestamp)
+) WITH comment='Index Table to look up alarms by acknowledged used by TIP WLAN CloudSDK';
+
+CREATE TABLE IF NOT EXISTS tip_wlan_keyspace.alarm_by_acknowledged_equipmentId (
+    customerId int,
+    equipmentId bigint,
+    alarmCode int, 
+    createdTimestamp bigint,   
+    acknowledged boolean,
+    
+    PRIMARY KEY ((customerId, acknowledged, equipmentId), alarmCode, createdTimestamp)
+) WITH comment='Index Table to look up alarms by acknowledged and equipmentId used by TIP WLAN CloudSDK';
+
+CREATE TABLE IF NOT EXISTS tip_wlan_keyspace.alarm_by_acknowledged_alarmCode (
+    customerId int,
+    equipmentId bigint,
+    alarmCode int, 
+    createdTimestamp bigint,   
+    acknowledged boolean,
+    
+    PRIMARY KEY ((customerId, acknowledged, alarmCode), equipmentId, createdTimestamp)
+) WITH comment='Index Table to look up alarms by acknowledged and alarmCode used by TIP WLAN CloudSDK';
+
+CREATE TABLE IF NOT EXISTS tip_wlan_keyspace.alarm_by_acknowledged_timestamp (
+    customerId int,
+    equipmentId bigint,
+    alarmCode int, 
+    createdTimestamp bigint,   
+    acknowledged boolean,
+    
+    PRIMARY KEY ((customerId, acknowledged), alarmCode, createdTimestamp, equipmentId)
+) WITH comment='Index Table to look up alarms by acknowledged and createdTimestamp used by TIP WLAN CloudSDK';
+
+
+# COPY tip_wlan_keyspace.alarm(customerId, equipmentId, alarmCode, createdTimestamp, acknowledged) TO 'alarmToAcknowledgedTable.csv';
+# COPY tip_wlan_keyspace.alarm_by_acknowledged(customerId, equipmentId, alarmCode, createdTimestamp, acknowledged) FROM 'alarmToAcknowledgedTable.csv';
+# COPY tip_wlan_keyspace.alarm_by_acknowledged_equipmentId(customerId, equipmentId, alarmCode, createdTimestamp, acknowledged) FROM 'alarmToAcknowledgedTable.csv';
+# COPY tip_wlan_keyspace.alarm_by_acknowledged_alarmCode(customerId, equipmentId, alarmCode, createdTimestamp, acknowledged) FROM 'alarmToAcknowledgedTable.csv';
+# COPY tip_wlan_keyspace.alarm_by_acknowledged_timestamp(customerId, equipmentId, alarmCode, createdTimestamp, acknowledged) FROM 'alarmToAcknowledgedTable.csv';
```

Comment:
these lines have to be wrapped into /* ... */ comment block, like the one below, otherwise cql shell will complain when ececuting cassandra schema script. There was a separate commit/pr about this in the master

Reply:
see commit 3ee708925f4193e3b52e4242cb802329e7b6f3bb